pontoon/base/views.py

```diff
@@ -353,31 +352,25 @@ def contributor(request, email, template='user.html'):
     return render(request, template, data)
 
 
-def contributors(request, template='users.html'):
+def contributors(request, period='alltime', template='users.html'):
```

Comment:
No need for `period` anymore, nothing is being passed in.

(Someday, but not today, that `template` kwarg is going to die soon.)